include/range/v3/view/any_view.hpp

```diff
@@ -498,19 +590,29 @@ namespace ranges
             }
 
             std::shared_ptr<detail::any_input_view_interface<Ref>> ptr_;
+            std::size_t size_;
         };
 
+#if RANGES_CXX_DEDUCTION_GUIDES >= 201606L
```

Comment:
We never use the raw numbers in code so readers don't have to remember what `201606L` means; use the symbolic constant `RANGES_CXX_DEDUCTION_GUIDES_17` instead.